metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -620,6 +620,9 @@ object ServerCommands {
       StartScalaCliServer,
       StopScalaCliServer,
     )
+
+  def allIds: Set[String] = all.map(_.id).toSet
```

Comment:
why not `val` 🤔

Reply:
Well, because I am quite blind and didn't realize what I wrote :sweat: